Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -156,6 +199,17 @@ def get_default_job_id(client: Client) -> CommandResults:
     )
 
 
+def get_feedback_field(params: Dict[str, Any]) -> CommandResults:
+    response = params.get("closing_reason_field")
+    readable_output = f'## {response}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Arcanna.FeedbackField',
+        outputs=response
```

Comment:
You did not add outputs for this command, so please add it or remove those lines.

Reply:
Added as arguemtn

Reply:
So the result looks like that.
Only a single value is put on the Arcanna.FeedbackField.
````
root:{} 2 items
Arcanna:{} 1 item
FeedbackField:closeReason
````

![feedbackfied](https://user-images.githubusercontent.com/6702878/131663029-14384f0b-9797-49e6-a729-9b4a97e59a22.png)